nautobot/extras/api/serializers.py

```diff
@@ -877,6 +902,39 @@ def get_display(self, obj):
         return obj.created.isoformat()
 
 
+#
+# Notes
+#
+
+
+class NoteSerializer(BaseModelSerializer):
+    url = serializers.HyperlinkedIdentityField(view_name="extras-api:note-detail")
+    user = NestedUserSerializer(read_only=True)
+    assigned_object_type = ContentTypeField(queryset=ContentType.objects.all())
+    assigned_object = serializers.SerializerMethodField(read_only=True)
```

Comment:
nit: `SerializerMethodField` is always read-only, don't need to explicitly state so.